app/commands/payments/stripe/payment_intent/create_for_payment.rb

```diff
@@ -8,7 +8,9 @@ def call
       customer: customer_id,
       amount: amount_in_cents,
       currency: 'usd',
-      setup_future_usage: 'off_session'
```

Comment:
Is this deprecated?

Reply:
it must be `nil` for this to work.

Reply:
If this isn't documented (and I can't find it), could you open a support request with Stripe to double-check this please.